crates/config/src/fuzz.rs

```diff
@@ -28,6 +28,8 @@ pub struct FuzzConfig {
     pub failure_persist_file: Option<String>,
     /// show `console.log` in fuzz test, defaults to `false`
     pub show_logs: bool,
+    /// Optional timeout (in seconds) for each property test
+    pub timeout: Option<u32>,
```

Comment:
Would `f32` seconds be useful?

Reply:
I think u32 is fine, @smartcontracts do you see any need for f32?

Reply:
u32 is fine for my use case, don't need perfect accuracy